spyder/widgets/mixins.py

```diff
@@ -22,9 +22,11 @@
 from qtpy.QtCore import QPoint, Qt
 from qtpy.QtGui import QCursor, QTextCursor, QTextDocument
 from qtpy.QtWidgets import QApplication, QToolTip
-from qtpy import PYQT5
+from qtpy import PYQT_VERSION
 
-if PYQT5:
+PYQT_VERSION = float(PYQT_VERSION)
+

```

Comment:
Maybe its cleaner to define

`PYQT55_VERSION = float(PYQT_VERSION) > 5.5`

And use that in the conditions below?